package.json

```diff
@@ -19,6 +19,9 @@
     "json:check": "npx ts-node --esm scripts/schema-check-files.ts src/data testdata/valid",
     "json:check:srcdata": "npx ts-node --esm scripts/schema-check-files.ts src/data",
     "json:check:testdata": "npx ts-node --esm scripts/schema-check-files.ts testdata/valid",
+    "json:generate:types": "npm run json:generate:types:metadata && npm run json:generate:types:timeseries",
+    "json:generate:types:metadata": "npx json-schema-to-typescript src/schema/pathwayMetadata.v1.json src/types/pathwayMetadata.v1.d.ts",
```

Comment:
Im considering just adding "prettier --write foo" here to ensure the generated types are always styled. 

Alternatively we can ignore these files in prettier config. 

Im not leaning super far in either direction really so happy to take feedback on board